tests/nest_tests/stdp_triplet_synapse_test.py

```diff
@@ -18,6 +18,7 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+import os
```

Comment:
we should move towards import statements ordered according to:

[import python standard library]

[import our own things, like nest and pynestml]

so in this case probably something like

```python
import numpy as np
import os
import pytest
```

Reply:
Do you also mean to order them alphabetically?